addons/terrabrush/Scripts/KeybindSettings.cs

```diff
@@ -0,0 +1,86 @@
+using Godot;
+using System.Text;
+
+namespace TerraBrush;
+
+[Tool]
+public partial class KeybindSettings : Godot.AcceptDialog
+{
+	[NodePath("VBoxContainer/Tree")]
+	private Tree _keybindList;
+	private TreeItem _root;
+	private KeybindManager _keybindManager;
+
+	private const int ADD_SHORTCUT = 1;
+	private const int ERASE_SHORTCUT = 2;
```

Comment:
I usually try to put constants before variables just because it should be one of the first thing you see in a file.

Also, I think it should be an enum :)

```suggestion
	private enum ShortcutType {
	    Add = 1,
	    Erase  = 2
	}
```